example.config.py

```diff
@@ -23,6 +23,9 @@
 # Modules we will NOT load
 disabled_modules = ['chat', 'wolframalpha', 'rss']
 
+# If whitelistonly, only these modules will load
```

Comment:
Please document this better, and leave this list empty in the default config. It might confuse new users.